types/core-js/index.d.ts

```diff
@@ -337,9 +337,9 @@ declare var log: Log;
 // #############################################################################################
 
 interface Dict<T> {
-    [key: string]: T;
-    [key: number]: T;
-    // [key: symbol]: T;
+    [key: string]: T | undefined;
```

Comment:
Why just these?  Why not (e.g.) `get`, below?

Reply:
some more changes added

Reply:
I don't see an update?

Reply:
my bad, sorry